src/evaluation/position_tables.h

```diff
@@ -198,12 +198,27 @@ constexpr auto generateBlackPassedPawnMaskTable()
     return data;
 }
 
+constexpr auto generateOutpostSquareMaskTable()
+{
+    using OutpostMaskTable = std::array<uint64_t, s_amountSquares>;
+    std::array<OutpostMaskTable, magic_enum::enum_count<Player>()> data {};
+
+    auto passedPawnTable = generatePassedPawnMaskTable();
+    auto fileMaskTable = generateFileMaskTable();
```

Comment:
can these be annotated constexpr? Might increase compilation time (unsure if it changes anything)